sdk/formrecognizer/ai-form-recognizer/src/generated/generatedClientContext.ts

```diff
@@ -30,10 +30,10 @@ export class GeneratedClientContext extends coreClient.ServiceClient {
       options = {};
     }
     const defaults: GeneratedClientOptionalParams = {
-      requestContentType: "application/json; charset=utf-8"
+      requestContentType: "application/json; charset=utf-8",
     };
 
-    const packageDetails = `azsdk-js-ai-form-recognizer/4.0.0-beta.1`;
+    const packageDetails = `azsdk-js-ai-form-recognizer/4.0.0-beta.2`;
```

Comment:
why this was not updated by the post release automation PR that should have been created after the beta.1 release? is the config for it set correctly in package.json?

Reply:
No idea.

Reply:
Apparently packageVersion needs to be "packageDetails."